doc/codi.txt

```diff
@@ -175,7 +181,20 @@ SHELL WRAPPER                                             *codi-usage-wrapper*
                  hi NonText ctermfg=0 |\
                  Codi $syntax" "$@"
              }
-<
+>
+VIRTUAL TEXT                                               *codi-virtual_text*
+
+If you're using NeoVim, by default it uses virtual text to print the output
+from the REPL, similar to other plugins used in other popular editors.
+
+Custom Highlight Group~
```

Comment:
Bump on this. I don't mind terribly if it stays here, though.

Reply:
Do you mean the whole `code-virtual_text` section or just the highlight group?

Reply:
Just the highlight group, since it's entirely about configuration.

I also noticed that the whole `codi-virtual_text` section is already mentioned in `codi-introduction` though, so I think the section should be removed after relocating the highlight group info.

Reply:
Done.